Lib/test/test_cprofile.py

```diff
@@ -151,23 +151,110 @@ def test_bad_descriptor(self):
 
 
 class TestCommandLine(unittest.TestCase):
+    def setUp(self):
+        self.test_script = tempfile.NamedTemporaryFile("w+", suffix=".py", delete=False)
+        self.test_script.write(textwrap.dedent("""
+            def simple_function():
+                sum(range(1000))
+
+            if __name__ == "__main__":
+                simple_function()
+        """))
+        self.test_script.close()
+
+    def tearDown(self):
+        os.unlink(self.test_script.name)
+
     def test_sort(self):
         rc, out, err = assert_python_failure('-m', 'cProfile', '-s', 'demo')
         self.assertGreater(rc, 0)
         self.assertIn(b"option -s: invalid choice: 'demo'", err)
 
+    def test_valid_sort_options(self):
+        for sort_opt in ['calls', 'cumulative', 'cumtime', 'filename',
+                         'ncalls', 'pcalls', 'line', 'name', 'nfl',
+                         'stdname', 'time', 'tottime']:
```

Comment:
```suggestion
        for sort_opt in ('calls', 'cumulative', 'cumtime', 'filename',
                         'ncalls', 'pcalls', 'line', 'name', 'nfl',
                         'stdname', 'time', 'tottime'):
```
Maybe just personal preference but I think tuples are better here

Reply:
Yes, using tuples makes more sense here.